Week3/Hotel.sql

```diff
@@ -0,0 +1,96 @@
+-- MySQL Workbench Forward Engineering
+
```

Comment:
Comments on ER diagram:

- I need more context to be able to understand the ER diagram. What is the business that you are building this database for? A hotel?